Gemfile

```diff
@@ -17,7 +17,7 @@ gem "image_processing"
 gem 'activemerchant', '>= 1.78.0'
 gem 'angular-rails-templates', '>= 0.3.0'
 gem 'awesome_nested_set'
-gem 'ransack', '~> 2.6.0'
+gem 'ransack', '~> 4.1.0'
```

Comment:
I think that we need a separate pull request just for this gem first. There are breaking changes.

* https://github.com/openfoodfoundation/openfoodnetwork/issues/10386